test/replicator.spec.js

```diff
@@ -76,7 +76,7 @@ Object.keys(testAPIs).forEach((IPFS) => {
 
         console.log(`writing ${logLength} entries to the log`)
         for (let i = 0; i < logLength; i++) {
-          await log2.append(`entry${i}`)
+          await log2.append(`entry${i}`, 10)
```

Comment:
This fix only works if we _don't_ use the second argument's default pointerCount of 1. Perhaps we can just make `sqrt(log.length)` the default? (pseudocode)

Reply:
not a bad idea! this kind of change could break some things so maybe its better suited to be introduced separately? especially since this change is unlikely to hurt anything.